__tests__/app.test.js

```diff
@@ -52,26 +53,28 @@ describe("task 4. GET /api/arcticles/:arcticle_id", () => {
       .get(`/api/articles/${article_id}`)
       .expect(200)
       .then(({ body }) => {
-        expect(body.articles).toEqual({
-          author: expect.any(String),
-          title: expect.any(String),
-          article_id: expect.any(Number),
-          body: expect.any(String),
-          topic: expect.any(String),
-          created_at: expect.any(String),
-          votes: expect.any(Number),
-        });
+        expect(body.articles).toEqual(
+          expect.objectContaining({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            body: expect.any(String),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+          })
```

Comment:
You're asking for a specific article here. So you can be afford to be specific with your testing. You know what the title should be, you know what the id should be.